content/v2.0/account-management/data-usage.md

```diff
@@ -18,15 +18,13 @@ right corner of your {{< cloud-name "short" >}} user interface (UI) and select *
 
 Find data usage information for the time frame specified in the drop-down at the top of the Usage page.
 
-- **Writes:** Total data in MB written to your {{< cloud-name "short" >}} instance.
-- **Reads:** Total data in MB sent as responses to queries from your {{< cloud-name "short" >}} instance.
-- **Query Duration:** Total time spent processing queries in seconds.
+- **Data In:** Total data in MB written to your {{< cloud-name "short" >}} instance.
+- **Data Out:** Total data in MB sent as responses to queries from your {{< cloud-name "short" >}} instance.
+- **Query Count:** Total number of events.
```

Comment:
"Events" seems really general. Is this the description they have on the new pricing page? What about "Total number of query executions?"

Reply:
I agree there could be more detail on this. Maybe summarize this description?

> Query Count is billed on the total number of individual query operations, which include queries, tasks (alerts, notifications) and Data Explorer activity. Any individual operation, such as the execution of an alert task, will be counted as one billable query operation. Refreshing a dashboard with multiple cells will incur multiple query operations. Any failed query/task executions are not counted.